index.html

```diff
@@ -2296,6 +2296,41 @@ <h1>Core Representations</h1>
 representations should be added here once that section has been written.
     </p>
 
+    <section>
+      <h2>Handling Properties</h2>
+      
+        <section>
+          <h2>Unknown Properties</h2>
+          <p>
+    When a DID Document is created by consuming a compliant representation, any properties
+    in the representation that are not known to the consumer MUST be added to the
```

Comment:
what is "not known to the consumer"... is this essentially "any properties"... I assume the consumer adds known properties as well.... can we define this more concretely?

Reply:
Agreed. Shouldn't this just be "preserve all properties"?

Reply:
"unknown" means "not a property that the consumer has been configured to understand (ie, core data model, registry, local extensions) and not a property that's known to be reserved (ie, registry, local extensions)". It'll have a data type in the source format that needs to map to a default data type in the abstract data model.

I think you're both conflating the JSON document with the abstract data model. They are not the same.

Reply:
what is a "local extension" thats not defined anywhere.... in JSON, we just add properties and they are preserved... if you are suggesting that somehow did documents deviate from what JSON developers will expect... i will continue to object.